source/common/http/conn_manager_impl.cc

```diff
@@ -284,7 +284,7 @@ void ConnectionManagerImpl::handleCodecException(const char* error) {
 
```

Comment:
can we avoid resetAllStreams both here and in doConnectionClose for consistent cleanup behavior or are there things in that block we don't want to pick up?

If so, let's comment why it's necessary to call here.

Reply:
Yes great catch, let me consolidate that now.